pkg/common/util/validation.go

```diff
@@ -0,0 +1,47 @@
+package util
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+// ValidateSpiffeID verify than Spiffe Id  is valid  according to SVID spec,
```

Comment:
I think this line should be written as: ValidateSpiffeID *verifies that a* Spiffe Id is valid  according to SVID spec,

Reply:
Maybe adding a link to https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md would be useful.

Reply:
done